File: task9/src/classesFigure/Rectangle.java
package classesFigure;

import classesFigure.base.Quadrilateral;

public class Rectangle extends Quadrilateral {
    public Rectangle(int sideA, int sideB) {
        if(isPositive(sideA) && isPositive(sideB)) {
            super.perimeter = (sideA + sideB) * 2;
            super.square = sideA * sideB;
            super.name = "Прямоугольник";
        } else {
            super.name = "";
        }
    }
}

File: task13/base/Rectangle.py
from base.Polygon import Polygon


class Rectangle(Polygon):
    __name = "пямоугольник"
    def __init__(self, sideA, sideB):
        p = (sideA + sideB) * 2
        s = sideA * sideB
        super().__init__(self.__name, s, p)